Makefile

```diff
@@ -90,16 +96,16 @@ check: cross build_e2e $(HOST_BUILD_DIR)/crc-embedder test cross-lint vendorchec
 
 .PHONY: install
 install: $(SOURCES)
-	go install -ldflags="$(LDFLAGS)" ./cmd/crc
+	go install -ldflags="$($(GOOS_LDFLAGS))" ./cmd/crc
```

Comment:
this is not working as expected, it's looking for a variable called `linux_LDFLAGS`, not `LINUX_LDFLAGS`